### PR TITLE
fix(examples/with-playwritght): use PORT env variable and fall back to 3000

### LINKED ISSUE #37952: Example works only with default port 3000
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```
    Operating System:
      Platform: darwin
      Arch: x64
      Version: Darwin Kernel Version 21.5.0: Tue Apr 26 21:08:22 PDT 2022; root:xnu-8020.121.3~4/RELEASE_X86_64
    Binaries:
      Node: 16.14.2
      npm: 8.5.0
      Yarn: 1.22.18
      pnpm: N/A
    Relevant packages:
      next: 12.1.7-canary.45
      react: 17.0.2
      react-dom: 17.0.2
```

### Which example does this report relate to?

with-playwright-app

### What browser are you using? (if relevant)

_No response_

### How are you deploying your application? (if relevant)

yarn test:e2e

### Describe the Bug

Playwright fails because it fails to detect when webserver starts.

### Expected Behavior

Playwright should succeed.

### To Reproduce

Set the port to 80 instead of leaving the default 3000, changing these 2 lines:
- `"dev": "next dev -p 80"` here <https://github.com/vercel/next.js/blob/canary/examples/with-playwright/package.json#L4>
- `3000` here <https://github.com/vercel/next.js/blob/canary/examples/with-playwright/playwright.config.ts#L19>

Run `DEBUG=pw:webserver yarn test:e2e`.

It fails because the `WebServer available event` is never fired.

### PR DESCRIPTION
**Pull request by thebergamo**
## Documentation / Examples

- [x] Related issues linked using `fixes #37952`
- [x] Make sure the linting passes by running `pnpm lint`
- [x] The examples guidelines are followed from [our contributing doc](https://github.com/vercel/next.js/blob/canary/contributing.md#adding-examples)

### CODE REVIEW

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
Thanks, there was an unnecessary change, let's revert that so we can merge this!

File: examples/with-playwright/README.md [hunk 1]
```diff
@@ -14,9 +14,17 @@ Execute [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packag
...
+**or**
+
+```bash
 yarn create next-app --example with-playwright with-playwright-app
-# or
+```
+
+**or**
+
+```bash
```
[the remark this answers is not in the document]
Comment (thebergamo) [reply examples/with-playwright/README.md:27]: Just for context, I got this spark while preparing this PR. 😛

[the remark this answers is not in the document]
Comment (balazsorban44) [reply examples/with-playwright/README.md:27]: We might do that, but let's do it in a different PR then, consistently across all examples. :+1:

**Comment by thebergamo** [comment]
@balazsorban44 `readme.md` changes reverted.

**Review by balazsorban44 — APPROVED** [review]

**Pull request merged**